sycl/test-e2e/ESIMD/unified_memory_api/atomic_update_slm.cpp

```diff
@@ -5,6 +5,7 @@
 // SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
 //
 //===----------------------------------------------------------------------===//
+// REQUIRES-INTEL-DRIVER: lin: 26918, win: 101.4953
```

Comment:
How did it fail with current CI driver? Did the test fails for one/few test cases? If Yes, it may be better to use isGPUDriverGE() function to guard those cases. Doing so will make it possible to test the majority of cases before 26918 reaches CI.
Otherwise (many cases fail), this solution is good.

Reply:
I narrowed the check with `isGPUDriverGE`.